src/sentry/utils/snuba.py

```diff
@@ -671,17 +671,15 @@ def get_related_project_ids(column, ids):
     return []
 
 
-def insert_raw(data):
-    data = json.dumps(data)
+def test_eventstream(data):
     try:
-        with timer('snuba_insert_raw'):
```

Comment:
`timer` incrs a metric and this method is only used in dev/test. Maybe I'm missing the point of this?